src/main/java/emt/gthandler/common/tileentities/machines/multi/generator/EMT_Large_Essentia_Gen.java

```diff
@@ -41,20 +41,8 @@ public EMT_Large_Essentia_Gen(String aName) {
 
     @Override
     public String[] getDescription() {
-        return new String[] { "Controller Block for the Large Essentia Generator.",
-                "Produces energy from magical essentia decay.", "Size(WxHxD): 7x4x7, Controller (Bottom, Center)",
-                "Top-Middle 5x1x5 layer are Muffler Hatches", "Middle 5x2x5 layer are Essentia Diffusion Cells",
-                "Everything from here on goes either in the bottom or on the sides", "1x Fluid Input Hatch",
-                "1x Essentia Input Hatch", "1x Dynamo Hatches", "1x Maintenance Hatch",
-                "The outer hull is made out of Magic Containing Casings",
-                "Produces 20x more than the singleblock generator per essentia",
-                "Causes magical and serious conventional pollution",
-                "Needs 1L Lubricant per second or will cause way more conventional pollution",
-                "Needs 10 " + EMTTextHelper.LIGHT_BLUE
-                        + "Aqua"
-                        + EMTTextHelper.LIGHT_GRAY
-                        + " cVis per second or will cause way more magical pollution",
-                "Added by EMT", "Made by bartimaeusnek" };
+        return new String[] { "Does not work, please don't use", "Marked for deprecation", "Added by EMT",
+                "Made by bartimaeusnek", "Terminated by Alastor" };
```

Comment:
this second line is not necessary. If it not exist anymore deprecated would be enough.

Reply:
Lmfao that's fair, it was a cheeky joke since Bart wrote his name on it, but I'll remove it